mr/src/test/java/org/apache/iceberg/mr/hive/TestHiveShell.java

```diff
@@ -186,6 +186,9 @@ private HiveConf initializeConf() {
     // Tez configuration
     hiveConf.setBoolean("tez.local.mode", true);
 
+    // Disable vectorization for HiveIcebergInputFormat
+    hiveConf.setBoolVar(HiveConf.ConfVars.HIVE_VECTORIZATION_ENABLED, false);
```

Comment:
I think this is fine, but I'm curious: do we plan to add vectorized writes? Or is this not something that we think is important?

Reply:
I think it is important, but no hard data on yet. Will work on read/write performance in the near future, so  would guess that vectorization will be something we check too.